Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs

```diff
@@ -38,7 +38,7 @@ internal async Task<List<Result>> SearchAsync(Query query, CancellationToken tok
             var quickaccessLinks = QuickAccess.AccessLinkListMatched(query, settings.QuickAccessLinks);
 
             if (quickaccessLinks.Count > 0)
-                results.AddRange(quickaccessLinks);
+                results.Union(quickaccessLinks);
```

Comment:
Distinct hasnt solved it yet, this change has been reverted. We probably need to write our own comparer

Reply:
Yeah I think you are right. By seeing the code of the equality comparator of Result class, it requires more than just title match. Maybe we still need to do that separately for plugins instead of distinct overall because we seldom will add two results that will be identical (they need to be equal at title, score, and highlight data) to the result list.

Reply:
yeah i agree. Perhaps we can create and add it to IPublicAPI so plugins can utilise the return distinct results themselves

Reply:
Well I don't think it should be included in IPublicAPI, because how they should filter the result should be decided by them. If we allow Equaltycomparater for that, I don't see the different between putting that in IPublicAPI and doing that by distinct or hashset.

Reply:
Let me add a customized add a customized equality comparator to do this job by comparing the subtitle (it is supposed to be the path) to remove duplicate for this one.